mypy/main.py

```diff
@@ -97,6 +97,21 @@ def type_check_only(sources: List[BuildSource], bin_dir: str, options: Options)
                        options=options)
 
 
+disallow_any_options = ['unimported']
+
+
+def disallow_any_argument_type(raw_options: str) -> List[str]:
+    flag_options = [o for o in raw_options.split(',') if len(o.strip()) > 0]
```

Comment:
If you want to explicitly disable all options for `--disallow-any` in config file you have to do 

```
disallow_any = 
```

This leads to call to `disallow_any_argument_type` with an empty string.
doing `raw_options.split(',')` if `raw_options` is empty, leads to a list with one empty element. And then this function raises an ArgumentError because an empty string is not a valid option.
But maybe I should put it in a comment in the code :)

I believe `argparse` already strips the whitespace but it might be safer to do it anyway.